assets/styles/layout/sidebar.scss

```diff
@@ -92,6 +92,11 @@ $sidebar-width: 250px;
       left: $sidebar-width;
       width: calc(100vw - #{$sidebar-width});
     }
+
+    .full-screen {
```

Comment:
how is this used?

Reply:
It practically reduces the screen size when the left sidebar is constantly open when it is on a bigger page. Without it, the text editor is behind the left sidebar. 
<img width="1440" alt="Screen Shot 2022-02-07 at 5 03 18 PM" src="https://user-images.githubusercontent.com/38061152/152879901-8dbed2e9-e5af-4fdb-bd7e-332d309a487c.png">